curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-algorithmic-thinking-by-building-a-dice-game/657e0c2c6a9d37705146f34d.md

```diff
@@ -21,6 +21,12 @@ You should have a `detectFullHouse` function.
 assert.isFunction(detectFullHouse);
 ```
 
+Your `detectFullHouse` function should have a parameter.
+
+```js
+assert.match(code, /const\s+detectFullHouse\s*=\s*(\(\s*\w+\s*\))\s*=>\s*/);
```

Comment:
This should work with a normal function declaration as well.

Reply:
@lasjorg updated PR.